doc/references.bib

```diff
@@ -1102,3 +1102,17 @@ @article{ZUO20102163
 	keywords = {Test–retest reliability, Intrinsic connectivity network, ICA, Dual regression, Resting state},
 	abstract = {Functional connectivity analyses of resting-state fMRI data are rapidly emerging as highly efficient and powerful tools for in vivo mapping of functional networks in the brain, referred to as intrinsic connectivity networks (ICNs). Despite a burgeoning literature, researchers continue to struggle with the challenge of defining computationally efficient and reliable approaches for identifying and characterizing ICNs. Independent component analysis (ICA) has emerged as a powerful tool for exploring ICNs in both healthy and clinical populations. In particular, temporal concatenation group ICA (TC-GICA) coupled with a back-reconstruction step produces participant-level resting state functional connectivity maps for each group-level component. The present work systematically evaluated the test–retest reliability of TC-GICA derived RSFC measures over the short-term (<45 min) and long-term (5–16 months). Additionally, to investigate the degree to which the components revealed by TC-GICA are detectable via single-session ICA, we investigated the reproducibility of TC-GICA findings. First, we found moderate-to-high short- and long-term test–retest reliability for ICNs derived by combining TC-GICA and dual regression. Exceptions to this finding were limited to physiological- and imaging-related artifacts. Second, our reproducibility analyses revealed notable limitations for template matching procedures to accurately detect TC-GICA based components at the individual scan level. Third, we found that TC-GICA component's reliability and reproducibility ranks are highly consistent. In summary, TC-GICA combined with dual regression is an effective and reliable approach to exploratory analyses of resting state fMRI data.}
 }
+
+@misc{abraham:hal-01093944,
```

Comment:
I think references are sorted. Could you move it to the right spot?

Reply:
I have moved this to the top and have added a 2 line space between it and the next reference. I noticed, however, that the file has a mix of 1 and 2 line spaces. Please let me know if I should change the current 2 line space to 1 line.